src/index.js

```diff
@@ -67,29 +76,86 @@ export function getTagValue(list, name) {
 		return;
 	}
 
-	let jwk = JSON.parse(Buffer.from(DEPLOY_KEY, 'base64').toString('utf-8'));
+	// Throw an error if both --eth and --pol are true
```

Comment:
This is not a very scalable method i think. An argument of `sig-type` or something that passes in values like would remove the need for these checks. Ideally it matches up with the arbundles signature config so we can map from their, but we can also use named values like `polygon` or `ethereum` to map to the right signers.

Reply:
I'm not sure I understand. We want deployments to default to Arweave unless something else is explicitly specified. Requiring a declaration of signer type in all cases would be a major breaking change. 

Can you clarify what you're requesting?

Reply:
I updated the -e and -p flags to use a single --sig-type flag instead, that defaults to arweave, and only valid signer types may be provided.